Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TurkishAirLinesHelper.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/19/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import Foundation
+
+class TurkishAirLinesHelper {
+    
+    enum GameDataStatus {
+        case schedule
+        case results
+    }
+    
+    let urls: Dictionary<GameDataStatus, String> = [
+        .schedule : "http://www.euroleague.net/euroleague/api/schedules?seasoncode=E2016",
+        .results : "http://www.euroleague.net/euroleague/api/results?seasoncode=E2016"
+    ]
+    
+    func getGamesTable() {
+        getSchedule()
+    }
+    
+    private func getSchedule() {
+        TurkishAirlinesApiClient
+            .getRequestFrom(
+                url: urls[.schedule]!,
+                parameters: [:],
+                headers: [:]){ [weak self] data ,error in
+                    if let xmlData = data, error == nil {
+                        
+                        self?.getResults()
+                    }
+                    else{
+                        
+                    }
+        }
+    }
+    
+    private func getResults() {
```

Comment:
Can we move private methods to extension that has all private methods and make them fileprivate